src/management-system-v2/lib/process-export/share-process-image-webshare-api.ts

```diff
@@ -0,0 +1,46 @@
+import { getSVGFromBPMN } from './util';
+import { getPNGFromSVG } from './image-export';
+import { BPMNCanvasRef } from '@/components/bpmn-canvas';
+
+/**
+ * share an image of a process or selected parts of a process using webshare api
+ *
+ * @param modeler the modeler to share the process image from
+ */
+export async function shareProcessImage(modeler: BPMNCanvasRef) {
+  let xml = await modeler.getXML();
```

Comment:
This is more or less the same code as `copyProcessImage`. It may be a good idea to consolidate by using a shared helper function